utils/utils.go

```diff
@@ -226,3 +226,14 @@ func VerifyFFMpegPath(path string) error {
 
 	return nil
 }
+
+// Removes the directory and makes it again. Throws fatal error on fail.
+func CleanupDirectory(path string) {
+	log.Traceln("Cleaning", path)
+	if err := os.RemoveAll(path); err != nil {
+		log.Fatalln(err)
+	}
+	if err := os.MkdirAll(path, 0777); err != nil {
+		log.Fatalln(err)
```

Comment:
Just because we're adding these errors to help users and not developers, could you add a little human readable blurb?  Maybe just a `log.Fatalln("Unable to remove/create directory, check ownership and permissions", err)` or something.  Other than that I think we're good to go!

Reply:
@gabek Done!